DOCS/tech-overview.txt

```diff
@@ -555,13 +555,13 @@ Condition variables
 -------------------
 
 They're used whenever a thread needs to wait for something, without nonsense
-like sleep calls or busy waiting. mpv uses the standard pthread API for this.
+like sleep calls or busy waiting. mpv uses the mp_thread API for this.
 There's a lot of literature on it. Read it.
 
 For initial understanding, it may be helpful to know that condition variables
-are not variables that signal a condition. pthread_cond_t does not have any
-state per-se. Maybe pthread_cond_t would better be named pthread_interrupt_t,
-because its sole purpose is to interrupt a thread waiting via pthread_cond_wait()
+are not variables that signal a condition. mp_cond does not have any
+state per-se. Maybe mp_cond would better be named mp_interrupt,
+because its sole purpose is to interrupt a thread waiting via mp_cond_wait()
```

Comment:
Now that we have our own interface, we aren't bound to copy pthread's naming. So this "maybe" can be turned into reality.

Reply:
It is not an interrupt. It is a way to signal that a new condition exists and that threads waiting for that condition can continue.

Reply:
Frankly speaking I have not read this `tech-overview.txt`. `mp_thread` is a thin pthread wrapper, which with all intention and purpose should work in similar way. Obviously it is limited to a subset of pthread and with higher level abstraction, like offloading the clock conversion from the caller.

I can take a look into updating this docs, but I was surprised in the first place that we have something like that in mpv, it is not really related specifically to mpv and mostly standard threading stuff that we are no place to explain.

Reply:
> Now that we have our own interface, we aren't bound to copy pthread's naming. So this "maybe" can be turned into reality.

I frankly don't agree with this `mp_interrupt` commentary. Also `cond` or `conditional variable` naming is standardized, not only by pthread, but anything else really. Including C++ `std::condition_variable` and so on. This make easier to understand what `mp_cond` represents. And it definitely is NOT interrupt.